R/fotmob_players.R

```diff
@@ -114,9 +115,13 @@
     "away_team_color" = .ppc(p, "teamData", "away", "color", n = n)
   )
 
-  rows$stats <- vector(mode = "list", length = nrow(stats))
-  for (i in 1:nrow(rows)) {
-    rows[i, ]$stats <- list(stats[1, ])
+  if (does_not_have_stats) {
+    rows$stats <- list(stats)
```

Comment:
this resolves the failing test [here](https://github.com/JaseZiv/worldfootballR/blob/e8daa4bbffae616ed28ecfba5961f544d3cdb81f/tests/testthat/test-fotmob.R#L482-L483)